src/test/java/com/wikia/webdriver/testcases/facebooktests/FacebookTests.java

```diff
@@ -70,20 +68,35 @@ public void linkWithExistingWikiaAccount() {
   }
 
   @Test
-  @RelatedIssue(issueID = "SOC-1492", comment = "No way to test manually")
   @UseUnstablePageLoadStrategy
   public void connectUsingUserPreferences() {
-    new RemoveFacebookPageObject(driver).removeWikiaApps(credentials.emailFB,
-        credentials.passwordFB).logOutFB();
+    new RemoveFacebookPageObject(driver).removeWikiaApps(emailFB, passwordFB).logOutFB();
 
     PreferencesPageObject prefsPage = new PreferencesPageObject(driver).open();
     prefsPage.loginAs(User.USER);
     prefsPage.selectTab(PreferencesPageObject.tabNames.FACEBOOK);
-    prefsPage.connectFacebook(credentials.emailFB, credentials.passwordFB);
+    prefsPage.connectFacebook(emailFB, passwordFB);
     prefsPage.verifyFBButtonVisible();
     prefsPage.logOut(wikiURL);
     SignUpPageObject signUp = new SignUpPageObject(driver).open();
     signUp.clickFacebookSignUp();
     prefsPage.verifyUserLoggedIn(credentials.userName);
   }
+
+  @BeforeMethod(alwaysRun = true)
+  public void setEnvironmentCredentials() {
```

Comment:
Test body doesn't seem to be the perfect place for environment related decision making block
@ludwikkazmierczak do you think this could be handled some other way ?
Also, each environment requires a separate e-mail ?